docs/gapic/v2/api.rst

```diff
@@ -1,5 +1,5 @@
-Client for Stackdriver Logging API
-==================================
+Client for Cloud Logging API
```

Comment:
@busunkim96 Is this branding change intended?

Reply:
Yes, Cloud Logging is a new brand name